client/packages/system/src/Vaccination/Components/VaccinationModal.tsx

```diff
@@ -79,7 +79,9 @@ export const VaccinationModal = ({
         const result = await saveVaccination(draft);
 
         if (result?.__typename === 'VaccinationNode') {
-          success(t('messages.vaccination-saved'))();
+          result?.invoice?.id && draft.createTransactions === true
```

Comment:
```suggestion
          result?.invoice?.id && draft.createTransactions
```

This is already evaluating a boolean (: So we don't need to check it against true